lib/fog/azurerm/requests/network/add_security_rules.rb

```diff
@@ -0,0 +1,152 @@
+module Fog
+  module Network
+    class AzureRM
+      # Real class for Network Request
+      class Real
+        def add_security_rules(resource_group, name, security_rules)
+          Fog::Logger.debug "Updating security rules in Network Security Group #{name}."
+          promise = @network_client.network_security_groups.get(resource_group, name)
+          result = promise.value!
+          nsg = result.body
+          security_rules_list = define_security_rules(security_rules)

```

Comment:
rename 
- security_rules_list to security_rules_objects
- rename sr to security_rule
- “define_security_rules” to create_security_rule_objects